lib/src/executable_options.dart

```diff
@@ -100,6 +104,24 @@ class ExecutableOptions {
   /// Whether to print the version of Sass and exit.
   bool get version => _options['version'] as bool;
 
+  /// Whether to run an interactive shell.
+  bool get interactive {
+    if (!(_options['interactive'] as bool)) return false;
+    if (_options.wasParsed('stdin') ||
+        _options.wasParsed('indented') ||
+        _options.wasParsed('load-path') ||
+        _options.wasParsed('style') ||
+        _options.wasParsed('quiet') ||
+        _options.wasParsed('help') ||
```

Comment:
`--help` takes precedence over all other options anyway, so there's no need to explicitly disallow it here.